modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -2,72 +2,213 @@
+
+[source,xml]
+----
+<flow name="scheduler-disallowConcurrentExecution-ex" >
+  <scheduler doc:name="Scheduler" disallowConcurrentExecution="true" >
+    <scheduling-strategy>
+      <fixed-frequency frequency="10" timeUnit="MILLISECONDS"/>
+    </scheduling-strategy>
+  </scheduler>
+  <!-- processors here -->
+</flow>
+----
+
+Each time a Scheduler skips its execution, Mule logs an `Execution skipped` message, for example (edited for readability):
+
+[source,logs]
+----
+INFO  2022-11-09 15:15:43,082 ...
+...scheduler.DefaultSchedulerMessageSource:
+Flow 'scheduler-disallowConcurrentExecution-ex' is already running and
+'disallowConcurrentExecution' is set to 'true'. Execution skipped.
+
+INFO  2022-11-09 15:15:43,083 ...
+...scheduler.DefaultSchedulerMessageSource:
+Flow 'scheduler-disallowConcurrentExecution-ex' is already running and
+'disallowConcurrentExecution' is set to 'true'. Execution skipped.
+----
+
+[[props_fixed_frequency]]
+=== Fixed Frequency (`<fixed-frequency/>`)
+
+The fixed frequency scheduling strategy accepts the following attributes:
+
+[%header,cols="1a,1a,1a,4a"]
+|===
+|Attribute | XML | Default | Description
+
 |*Frequency*
-|The frequency at which the Scheduler triggers the flow. You select the unit of time in the *Time unit* field. Setting the value to `0` or a negative value causes the default value to be used.
-|
-1000
+| `frequency`
+| `1000`
+|The frequency at which the Scheduler triggers the flow given the `timeUnit` setting.
 
 | *Start Delay*
-| The amount of time to wait before triggering the flow for the first time after the application is started. This value is expressed in the same unit of time as the frequency.
-| 0
+| `startDelay`
+| `0`
+| The amount of time to delay execution of the Scheduler for the first time after the application starts. This property uses the same `timeUnit` as the frequency.
 
 | *Time Unit*
-| The time unit for the values of *Frequency* and *Start Delay*. The possible values are:
+| `timeUnit`
+| `MILLISECONDS`
+| One of the following time units for the values of `frequency` and `startDelay`:
+
+`MILLISECONDS`, `SECONDS`, `MINUTES`, `HOURS`, or `DAYS`
+
+|===
+
+This strategy delays execution of the Scheduler for 10 seconds and then executes every 10 seconds:
+
+[source,xml]
+----
+<scheduler doc:name="Scheduler" >
+  <scheduling-strategy >
+    <fixed-frequency frequency="10" timeUnit="SECONDS" startDelay="10"/>
+  </scheduling-strategy>
+</scheduler>
+----
+
+[[props_cron_expressions]]
+=== Cron (`<cron/>`)
+
+For more complex scheduling strategies, you can use a cron expression.
 
-MILLISECONDS, SECONDS, MINUTES, HOURS, DAYS
-| MILLISECONDS
+[%header,cols="1a,1a,4a"]
 |===
+|Attribute |XML |Description
 
+| *Expression*
+| `expression`
+| Use a cron expression for when to do a trigger. There is no default cron expression. For guidance with expressions, see <<cron_expressions>>. Example: `expression="1 1 1 1,7 * ?"`
```

Comment:
"do a trigger" is not clear.

"when to do a trigger" > when to trigger the flow / when to execute a trigger???

Reply:
done